tests/foreman/ui/test_discoveredhost.py

```diff
@@ -19,10 +19,9 @@
 from fauxfactory import gen_string
 from nailgun import entities
 
+from robottelo import ssh
```

Comment:
The previous one was correct:

```suggestion
from robottelo.utils import ssh
```